ecies.c

```diff
@@ -19,14 +19,6 @@ const char key[] = \
 "nzwv/wzEUsNPrLSUfMq1dYvt3hk=\n" \
 "-----END PRIVATE KEY-----\n";
 
-void RAND_init(void) {
```

Comment:
Yes, this is usual OpenSSL mess and controversial doc and code content. As you may remember great yada yada about heartbleed bug in OpenSSL and all these talks about its need to make OpenSSL more strict and clear library. But it seems nothing much has changed since 2014 and I think I'd better go back to the Crypto++.

On Windows with the OpenSSL version `OpenSSL_1_1_0f` I can assure you: 
This code works:
```
    if (!initialized) {
        RAND_poll();
        initialized = 1;
    }
```
and `RAND_poll` calls [CryptGenRandom](https://msdn.microsoft.com/en-us/library/windows/desktop/aa379942%28v=vs.85%29.aspx) and then `RAND_add`